RecoHI/HiTracking/python/HITrackingRegionProducer_cfi.py

```diff
@@ -88,3 +88,15 @@
 HiTrackingRegionFactoryFromSTAMuonsBlock.MuonTrackingRegionBuilder.vertexCollection = cms.InputTag("hiSelectedVertex")
```

Comment:
@echapon @yetkinyilmaz 
Could you please comment if `RecoHI/HiMuonAlgos/python/HiRegitMuonLowPtTripletStep_cff.py` is still needed (it looks unused currently)? Thanks.